ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/event-handlers.md

```diff
@@ -35,7 +46,32 @@
         delay: <int>
 
         # Configure event trigger condition.
-        trigger: <str; "on-boot" | "on-logging" | "on-startup-config">
+        trigger: <str; "on-boot" | "on-logging" | "on-startup-config" | "on-maintenance">
```

Comment:
I see below options, not sure if we need to add all. Ignore if not mentioned in actual task.
```
wan1-dc1(config-handler-test1)#trigger ?
  on-boot            Trigger condition occurs on system boot
  on-config          Trigger the event handler when it is configured
  on-counters        Trigger condition occurs on evaluating statistical counters
  on-intf            trigger condition occurs on specified interface changes
  on-logging         Trigger condition occurs when regex match any log message
  on-maintenance     Trigger condition occurs on maintenance operation
  on-startup-config  trigger condition occurs on startup config changes
  vm-tracer          Trigger condition occurs on VmTracer events
  ```

Reply:
This PR is for `on-maintenance` only